app-starters-common/pom.xml

```diff
@@ -24,4 +24,12 @@
 		<module>app-starters-twitter-common</module>
 	</modules>
 
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-java-dsl</artifactId>
+			<version>${spring-integration-java-dsl.version}</version>
+		</dependency>
```

Comment:
The apps that we are generating using the starters make use of Initializr API directly and they use a specific version of boot (1.3.3 at the moment). So, the pre built apps that we provide and the apps generated through http://start-scs.cfapps.io/ both come with a specific version of boot. Of course, the users are free to change this for their needs. Another reason is not to include the DSL implicitly is that not all the apps or starters need the Java DSL, not a big deal to include it anyway, but unnecessary I think.